utils/release_tracking.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import re
+import os
+import subprocess
+import sys
+
+
+def run_cargo_build(path):
+    print(f"cargo build, path: {path}")
+    command = f'cargo build'
```

Comment:
That's why I put the `command.split()`
I first make a string literal for the command so that it's easier to read for us developers, and then I split it into a list for `check_output`. It's easier to read this way :)

Reply:
Done